firebase-firestore/src/main/java/com/google/firebase/firestore/remote/AbstractStream.java

```diff
@@ -245,14 +247,6 @@ public void start() {
     // Note that Starting is only used as intermediate state until onOpen is called asynchronously,
     // since auth handled transparently by gRPC
```

Comment:
I think this comment is kinda' stale and should probably be removed  (sorry I didn't notice before).

Reply:
Removed